tracer/test/snapshots/AspNetCoreMvc21WrongMethodTests._.verified.txt

```diff
@@ -0,0 +1,26 @@
+﻿[
+  {
+    TraceId: Id_1,
+    SpanId: Id_2,
+    Name: aspnet_core.request,
+    Resource: POST /,
+    Service: Samples.AspNetCoreMvc21,
+    Type: web,
+    Tags: {
+      component: aspnet_core,
+      env: integration_tests,
+      http.method: POST,
+      http.request.headers.host: localhost:00000,
+      http.status_code: 404,
```

Comment:
I think this was a change of behaviour from .NET Core 2.x to .NET Core 3.x - in earlier versions an incorrect method is treated as a 404, in later versions they changed that to a 405. So this is correct.